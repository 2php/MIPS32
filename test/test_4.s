.set noreorder
#Write com function test
start:
  .text 0
  la $s0, 0xbfd003f8
  la $s1, 0xbfd003fc
  li $a0, 0x23
  jal write_com
  nop
  li $a0, 0x24
  jal write_com
  nop
  li $a0, 0x25
  jal write_com
  nop
  break

write_com:
  li $t1, 1
  loop:
  lw $t0, 0($s1)
  andi $t0, $t0, 1
  bne $t0, $t1, loop
  sw $a0, 0($s0)
  jr $ra
