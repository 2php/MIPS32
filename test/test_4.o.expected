0x23
0x24
0x25
